rskj-core/src/main/resources/expected.conf

```diff
@@ -220,6 +220,7 @@ rpc = {
                 enabled = <enabled>
                 bind_address = <bind_address>
                 port = <port>
+                server_write_timeout = <timeout>
```

Comment:
Same here if using "seconds" in the variable name.

Reply:
renamed